plugins/woocommerce/tests/e2e-pw/tests/merchant/create-cart-block.spec.js

```diff
@@ -23,6 +24,10 @@ test.describe(
 		} ) => {
 			await goToPageEditor( { page } );
 
+			// Dismiss "Choose a pattern" if present (Pressable)
+			// eslint-disable-next-line playwright/no-conditional-in-test
```

Comment:
I'm not sure this comment disabling eslint is still required. 
Actually, the first both comment is also unnecessary, the closeChoosePatternModal function name is pretty self explanatory IMO.

Reply:
Agree, the first comment left after the first change (comment [here](https://github.com/woocommerce/woocommerce/pull/51014#discussion_r1734598246)). I'll remove them.

Reply:
Done [ed051f4](https://github.com/woocommerce/woocommerce/pull/51014/commits/ed051f48680f9a706925f5f5a8e2babcf4f57d24)